.github/workflows/buf.yml

```
@@ -0,0 +1,20 @@
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  steps:
+    - uses: actions/checkout@v2
+    - uses: bufbuild/buf-setup-action@v0.5.0
```

We can use `v0.6.0` here:
```suggestion
    - uses: bufbuild/buf-setup-action@v0.6.0
```

I'll update the out-of-date README.md in `buf-push-action`.